tex/iii.1/_0131_0132c.tex

```diff
@@ -100,8 +100,6 @@
 працюючого дрібного селянина, або контролю асоційованих
 виробників.
 
-\pfbreak
```

Comment:
We can't remove pfbreak if they are already in original file, and can't add them